README.md

```diff
@@ -1,19 +1,58 @@
-Check this [presentation](https://docs.google.com/presentation/d/1LnjIQT7tTrk7V7zK8xPE4LW-R5lJbAAPvDVEvPU6_xA/edit) for more information.  
+This tutorial shows some scenarios related to the Cilium and Spire integration.
+The image below represents the summary of the actions performed in each of them.
+
+<img src="imgs/background.png" alt="drawing" width="800"/>
 
 ## First steps
 
-- Download repository dependencies: `go vendor`
-- Setup cilium development environment
-- Checkout the branchs
-	- cilium: kinvolk:mauricio/spiffe-integration-poc-part2 
-	- cilium-proxy: kinvolk:mauricio/spiffe-integration-poc-part2 
-- Compile and install `cilium` and `cilium-proxy` 
-- Deploy `spire-agent` and `spire-server`: `kubectl -f spire.yaml`
+Download repository dependencies:
+
+```
+go vendor
+```
+
+Create minikube cluster:
+
+```
+minikube start --network-plugin=cni --memory=4096
+minikube ssh -- sudo mount bpffs -t bpf /sys/fs/bpf
+```
+
+Deploy manifest (cilium + spire + dependencies):
+
+```
+kubectl apply -f cilium-spire.yaml
+```
+
+Check pods status:
```

Comment:
`Check the status of the all the pods.  The spire-control-plane (spire-agent and spire-server) should be Running as well as the cilium-control-plane.`
```suggestion
Check the status of the all the pods.  The spire-control-plane (spire-agent and spire-server) should be Running as well as the cilium-control-plane.
```